### PR TITLE
[one-cmds] Installation infrastructure for onnx_legalizer tests

### PR DESCRIPTION
**Pull request by binarman**
This PR adds:
- related CMake and prepare_test_materials sections.
- installs onnx_runtime in one-prepare-venv

Related issue: #8395

ONE-DCO-1.0-Signed-off-by: Alexander Efimov <a.efimov@samsung.com>

### CODE REVIEW

**Comment by binarman** [comment]
It is draft, because I want to add some negative cases, which affects `prepare_test_materials.sh`

**Review by seanshpark — DISMISSED** [review]
LGTM

**Comment by seanshpark** [comment]
@mhs4670go , if we need to update debian package file, please add comment

File: compiler/one-cmds/tests/prepare_test_materials.sh [hunk 1]
```diff
@@ -91,6 +91,29 @@ if [[ ! -s "onnx_conv2d_conv2d.onnx" ]]; then
...
+   [ ! -s "RNN-noinit.onnx" ] ||
+   [ ! -s "LSTM.onnx" ] ||
+   [ ! -s "LSTM-bi.onnx" ] ||
+   [ ! -s "LSTM-noinit.onnx" ] ||
+   [ ! -s "LSTM-nobias.onnx" ]; then
+    rm -rf test_onnx_recurrent_models.zip
+    wget https://github.com/Samsung/ONE/files/8067909/test_onnx_recurrent_models.zip
+    unzip test_onnx_recurrent_models.zip
+    # https://github.com/Samsung/ONE/issues/8395#issuecomment-1040072097
+fi
```
Comment (mhs4670go) [inline compiler/one-cmds/tests/prepare_test_materials.sh:107]: Maybe you can add an array for these files.
```suggestion

declare -a RECURRENT_MODELS=("RNN.onnx" "RNN-nobias.onnx" "RNN-relu.onnx" "RNN-bi.onnx")

## now loop through the above array
for i in "${RECURRENT_MODELS[@]}"
do
   if [ ! -s "$i" ]; then
   # do something
   break
   fi
done
```

**Review by mhs4670go — APPROVED** [review]
LGTM

**Review by seanshpark — APPROVED** [review]
LGTM

**Pull request merged**